activemodel/lib/active_model/locale/en.yml

```diff
@@ -10,10 +10,10 @@ en:
       inclusion: "is not included in the list"
       exclusion: "is reserved"
       invalid: "is invalid"
-      confirmation: "doesn't match %{attribute}"
+      confirmation: "doesn’t match %{attribute}"
       accepted: "must be accepted"
-      empty: "can't be empty"
-      blank: "can't be blank"
+      empty: "can’t be empty"
```

Comment:
Wouldn't this be a breaking change?

Reply:
Can you expand how exactly? I agree that the outward result is now slightly different, but would you also consider fixing a spelling mistake or a grammar mistake a breaking change?

If that is the case, what is the appropriate way to improve user facing text? I'm open to taking the necessary path.

Reply:
I'm not 100% sure, but if this changes the error messages after people upgrade, that may break people's tests.

I'm imagining this case:

```ruby
test "some validation" do
  user = User.new(name: "")
  user.valid?
  assert user.errors.include?("can't be blank") # fails because the quote changed
end
```

Reply:
I agree that is a possibility, but is the precise error text a guarantee of the API such that we can't make spelling, grammar or typographical improvements?

If this landed for Rails 7.0 instead of a 6.* point release, would it resolve that concern?

Reply:
I'm not a core rails developer, so I don't know how they will proceed. I just pointed out that this may be a breaking change. Let's wait for more feedback, ok?